src/hooks/useMarketForm.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from 'react';
+
+const useMarketoForm = (munchkinId, id, pubishableKey) =>
+  useEffect(() => {
+    window.MktoForms2.loadForm('//app-abj.marketo.com', munchkinId, id);
+
+    const pollForDefinition = function (scope, varname, callback) {
```

Comment:
```suggestion
    const pollForDefinition = (scope, varname, callback) => {
```

[nit] we typically use the anonymous arrow function syntax and tend to avoid the `function` keyword when possible.